### ISSUE TITLE
tail -F /dev/random

### ISSUE THREAD

**Issue by xme** [PhonePe/mantis#14]
Hi *,

Running a "onboard" operation with "productsecurity" profile.
It became slower and slower and I detected the following command running 100% of the CPU (and up for 32 hours!):

`tail -F /dev/random
`

The parent process is:

`/usr/bin/containerd-shim-runc-v2 -namespace moby -id 8da7d4f4215c919e01e68abffa7079c73839e0136961bd1aadb00b6ba23fcaaf -address /run/containerd/containerd.sock
`
Any idea how to fix this?
The server is a freshly installed Ubuntu 22.04 LTS

**Comment by 0xbharath** [comment]
The usage of `tail -F /dev/random` was a hack-ish way to keep the Mantis container from exiting. Will fix this more gracefully.

However, I don't think this command would cause the CPU usage spike. Have you been running Mantis scans during this time? Mantis scans tend to be CPU intensive and could have been the cause for the spike.

**Comment by xme** [comment]
Ok, I understand the purpose of this command. I see now that one of my CPUs is at 100% and no scan is running. This is not super critical but it hurts me to see so many CPU cycles for nothing :-)

**Comment by isubhampadhi56** [comment]
I got the issue the tail command cause busy waiting. So it's better to use sleep command to keep container alive. While debugging this issue i found another small issue with Dockerfile of dashboard for which build process was failing for me. So I also fixed the update command under that file
![Screenshot from 2024-04-23 12-48-19](https://github.com/PhonePe/mantis/assets/25567957/3fb45c79-3a55-4643-ae57-84aa6318dcba)

PR Link - https://github.com/PhonePe/mantis/pull/18
